student.rb

```diff
@@ -1,17 +1,17 @@
+require_relative 'common_point'
+
 class Student
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
These look similar to stuff in the teacher classes... how could you refactor it up?